src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -126,9 +128,20 @@ public class SnowflakeSinkConnectorConfig {
   public static final int SNOWPIPE_FILE_CLEANER_THREADS_DEFAULT = 1;
 
   // This is the streaming max client lag which can be defined in config
+  public static final String SNOWPIPE_STREAMING_ENABLE_SINGLE_BUFFER =
+      "snowflake.streaming.enable.single.buffer";
+
+  public static final boolean SNOWPIPE_STREAMING_ENABLE_SINGLE_BUFFER_DEFAULT = false;
   public static final String SNOWPIPE_STREAMING_MAX_CLIENT_LAG =
       "snowflake.streaming.max.client.lag";
 
+  public static final String SNOWPIPE_STREAMING_MAX_CHANNEL_SIZE =
+      "snowflake.streaming.max.channel.size";
+  public static final long SNOWPIPE_STREAMING_MAX_CHANNEL_SIZE_DEFAULT = 128 * 1024 * 1024;
```

Comment:
this is a bit confusing, max channel count size? or size of data in it?

what will this do?